tools/onnx2tensorrt.py

```diff
@@ -0,0 +1,246 @@
+        os.makedirs(save_dir, exist_ok=True)
+    save_trt_engine(trt_engine, trt_file)
+    print(f'Successfully created TensorRT engine: {trt_file}')
+
+    if verify:
+        inputs = _prepare_input_img(model_type=model_type, img_path=img_path, config=config)
+
+        imgs = inputs['imgs']
+        img_list = [imgs.unsqueeze(0)]
+
+        if max_shape[0] > 1:
+            # concate flip image for batch test
+            flip_img_list = [_.flip(-1) for _ in img_list]
+            img_list = [
+                torch.cat((ori_img, flip_img), 0)
+                for ori_img, flip_img in zip(img_list, flip_img_list)
+            ]
+
+        # Get results from ONNXRuntime
+        ort_custom_op_path = get_onnxruntime_op_path()
+        session_options = ort.SessionOptions()
+        if osp.exists(ort_custom_op_path):
+            session_options.register_custom_ops_library(ort_custom_op_path)
+        sess = ort.InferenceSession(onnx_file, session_options)
+        sess.set_providers(['CPUExecutionProvider'], [{}])  # use cpu mode
+        onnx_output = sess.run(['output'],
+                               {'input': img_list[0].detach().numpy()})[0][0]
+
+        # Get results from TensorRT
+        trt_model = TRTWraper(trt_file, ['input'], ['output'])
+        with torch.no_grad():
+            trt_outputs = trt_model({'input': img_list[0].contiguous().cuda()})
+        trt_output = trt_outputs['output'][0].cpu().detach().numpy()
+
+        if show:
+            onnx_visualize = onnx_output.transpose(1, 2, 0)
+            onnx_visualize = np.clip(onnx_visualize, 0, 1)[:, :, ::-1]
+            trt_visualize = trt_output.transpose(1, 2, 0)
+            trt_visualize = np.clip(trt_visualize, 0, 1)[:, :, ::-1]
+
+            cv2.imshow('ONNXRuntime', onnx_visualize)
+            cv2.imshow('TensorRT', trt_visualize)
+            cv2.waitKey()
+
+        np.testing.assert_allclose(
+            onnx_output, trt_output, rtol=1e-03, atol=1e-05)
+        print('TensorRT and ONNXRuntime output all close.')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert MMSegmentation models from ONNX to TensorRT')
+    parser.add_argument('config', help='Config file of the model')
+    parser.add_argument(
+        'model_type',
+        help='what kind of model the config belong to.',
+        choices=['inpainting', 'mattor', 'restorer', 'synthesizer'])
+    parser.add_argument(
+        'img_path', type=str, help='Image for test')
+    parser.add_argument('onnx_file', help='Path to the input ONNX model')
+    parser.add_argument(
+        '--trt-file', type=str, help='Path to the output TensorRT engine', default='tmp.trt')
+    parser.add_argument(
+        '--max-shape',
+        type=int,
+        nargs=4,
+        default=[1, 3, 512, 512],
+        help='Maximum shape of model input.')
+    parser.add_argument(
+        '--min-shape',
+        type=int,
+        nargs=4,
+        default=[1, 3, 32, 32],
```

Comment:
Why the default min shape has such a small height, width to 32?

Reply:
There is a picture in Set5 with shape of 63*63